README.md

```diff
@@ -187,7 +187,6 @@ Docker containers:
 - 3rd [Top Offensive Security Open Source Projects](https://awesomeopensource.com/projects/offensive-security) (2022-01-27)
 - Recent mentions on social medias: [Social-searcher](https://www.social-searcher.com/social-buzz/?q5=https%3A%2F%2Fgithub.com%2Fnoraj%2FOSCP-Exam-Report-Template-Markdown)
 - Articles:
-  - ~~[Stress-free OSCP report making](https://craigunder.me/stress-free-oscp-report-making/)~~ by Craig Underhill (2020-04-06)
```

Comment:
I would rather edit the link and add an archive rather than deleting.

```md
Stress-free OSCP report making by Craig Underhill (2020-04-06) ([Raw Archive](https://web.archive.org/web/20210729100632/https://craigunder.me/stress-free-oscp-report-making/) or Firefox Reader link `about:reader?url=https://web.archive.org/web/20210729100632/https://craigunder.me/stress-free-oscp-report-making/` (better look, more readable))
```

Reply:
Understood.